resource/lvm/fs/preparer.go

```diff
@@ -0,0 +1,33 @@
+package fs
+
+import (
+	"github.com/asteris-llc/converge/resource"
+	"github.com/asteris-llc/converge/resource/lvm/lowlevel"
+
+	"strings"
+)
+
+// Preparer for LVM FS Task
+type Preparer struct {
+	Device     string   `hcl:"device",required:"true"`
+	Mount      string   `hcl:"mount",required:"true"`
+	Fstype     string   `hcl:"fstype",reqired:"true"`
```

Comment:
It might be nice to add allowed values here to get nicer error messages; I know we can't exhaustively list every possible filesystem though so maybe not.  What do you think?